localization/vision_common/CMakeLists.txt

```diff
@@ -29,9 +29,9 @@ find_package(catkin2 REQUIRED COMPONENTS
 # Find GTSAM
 find_package(GTSAM REQUIRED)
 
-# Find OpenCV3
+# Find OpenCV
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../../cmake")
-find_package(OpenCV331 REQUIRED)
+find_package(OpenCV4 REQUIRED)
```

Comment:
yeah, it's not clear... OpenCV4 is the local cmake that includes the xfeatures2d. not all packages need it, so for the ones that don't I ust included the system one

Reply:
Oh interesting. Is there a reason not to just always use our modified opencv?

Reply:
no :)

Reply:
Hmm... What do you think now? I suspect it would be easy to switch if necessary, and it's probably not worth the potential confusion.

Reply:
maybe change our cmake to FindOpenCV4WithXFeatures.cmake to make it explicit it's an astrobee spacific include?

Reply:
ok so because we have to add this line:

LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../../cmake")

every time we add our own cmake I opted for the name change to OpenCV4WithXFeatures since it was simpler